src/components/MyAvatarMenu/MyAvatarMenu.tsx

```diff
@@ -70,7 +60,7 @@ export const MyAvatarMenu = () => {
             }}
           >
             <Link href="#">
-              <ApeAvatar profile={myProfile} className={classes.avatarButton} />
+              <Avatar path={myProfile.avatar} size="medium" />
```

Comment:
When showing the avatar for a user profile, you should also set the `name` property, so that if the profile doesn't have an avatar image, we can show the initials of the name instead